hack/unwanted-dependencies.json

```diff
@@ -55,6 +55,7 @@
       "github.com/imdario/mergo": "see https://github.com/kubernetes/kubernetes/issues/107499",
       "github.com/influxdata/influxdb1-client": "db/datastore clients should not be required",
       "github.com/json-iterator/go": "refer to #105030",
+      "github.com/klauspost/compress": "unreviewable assembly code, `prometheus/client_golang` should use stdlib instead",
```

Comment:
See notes in https://github.com/kubernetes/kubernetes/pull/130569#issuecomment-2700670214